pkg/parser/antlr/FqlParser.g4

```diff
@@ -6,8 +6,16 @@ program
     : body
     ;
 
+useExpression
+    : useExpressionAs
+    ;  
+
+useExpressionAs
+    : Use As
+    ;
+
 body
-    : (bodyStatement)* bodyExpression
+    : useExpression
```

Comment:
You need create ``head`` section. And add it to ``program``.

Reply:
It's works local now! Thanks!